scripts/source-maintenance.sh

```diff
@@ -40,6 +40,26 @@ fi
 COPYRIGHT_YEARS=`date +"1996-%Y"`
 echo "s/1996-2[0-9]+ The Squid Software Foundation and contributors/${COPYRIGHT_YEARS} The Squid Software Foundation and contributors/g" >>boilerplate_fix.sed
 
+updateifchanged()
```

Comment:
Nit: Please use camelCase or snake_case for new names.

Reply:
done